File: lua/test/exists_test.lua
-- TiktokEngagementBot SDK exists test

local sdk = require("tiktok-engagement-bot_sdk")

describe("TiktokEngagementBotSDK", function()
  it("should create test SDK", function()
    local testsdk = sdk.test(nil, nil)
    assert.is_not_nil(testsdk)
  end)
end)

File: ts/test/exists.test.ts

import { test, describe } from 'node:test'
import { equal } from 'node:assert'


import { TiktokEngagementBotSDK } from '..'


describe('exists', async () => {

  test('test-mode', () => {
    const testsdk = TiktokEngagementBotSDK.test()
    equal(testsdk instanceof TiktokEngagementBotSDK, true,
      'TiktokEngagementBotSDK.test() must return a client synchronously')
  })

})
